### ISSUE TITLE
downloaded this connector and it failed to build

### ISSUE THREAD

**Issue by davidradl** [odpi/egeria-connector-xtdb#430]
I forked and cloned this connector. I did a maven clean install and it failed with 👍

[ERROR] Failed to execute goal on project egeria-connector-crux: Could not resolve dependencies for project org.odpi.egeria:egeria-connector-crux:jar:3.1-SNAPSHOT: The following artifacts could not be resolved: org.odpi.egeria:audit-log-framework:jar:3.1-SNAPSHOT, org.odpi.egeria:repository-services-apis:jar:3.1-SNAPSHOT, org.odpi.egeria:open-connector-framework:jar:3.1-SNAPSHOT, org.odpi.egeria:repository-services-implementation:jar:3.1-SNAPSHOT, org.odpi.egeria:connector-configuration-factory:jar:3.1-SNAPSHOT, org.odpi.egeria:open-metadata-types:jar:3.1-SNAPSHOT: org.odpi.egeria:audit-log-framework:jar:3.1-SNAPSHOT was not found in https://oss.sonatype.org/content/repositories/snapshots during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of ossrh-snapshots has elapsed or updates are forced -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.

I changed the 3.1-SNAPSHOT to 3.2 in the pom file and it compiled cleanly.

**Comment by cmgrote** [comment]
3.1 release of the connector went out more than a year ago (September 2021) -- I'm not sure how long Maven Central keeps snapshots around, but probably not that long?

Any particular reason you're trying to build against such an old version, and not using the release but a snapshot?

(Any chance you're using a very old fork of the repository before it was renamed xtdb and that is causing some stale artifices?)

**Comment by davidradl** [comment]
Hi Chris, thanks  for you comments. I was trying to find a build that completed and the old 3.2 one did; but the latest released Egeria code did not build for me. 

I will clear out my .m2 in case there is something stale locally is causing this.

**Comment by davidradl** [comment]
You are right @cmgrote - I did not realise I was building against an old version
